definitions/push-images/push-images-pipeline.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: push-images-pipeline
+spec:
+  tasks:
+    - name: skopeo-copy-task
+      params:
+        - name: srcImageURL
+          value: "docker://quay.io/repository/hacbs-release/skopeo-copy-source"
+          description: Temp hardcoded POC values
```

Comment:
This is where the parameter values are passed to the task (as this is the pipeline definition, not task definition). Why are there parameter descriptions defined here?